en/docs/references/analyze-performance.md

```diff
@@ -0,0 +1,58 @@
+# Performance Analysis
+
+The performance Analyzer tool is an AI-based tool that generates intelligent estimations of the performance of service-based applications based on historical data. To generate the forecasts, it uses advanced Machine Learning techniques.
```

Comment:
instead service-based applications, shall we use the phrase `integration-based services and APIs`?

Reply:
Changed as suggested.